library/default-effects-extra.scd

```diff
@@ -380,4 +380,37 @@ DEFAULT EFFECTS EXTRA
 			).tanh; // .tanh is used as a crude limiter here beacause sometimes this ugen goes crazy
 		ReplaceOut.ar(out, signal)
 	}, [\ir, \ir]).add;
+
+
+    // DJ filter, a low pass filter for the first half of the range, and a high pass for the rest.
+    ~dirt.addModule('dj-filter', { |dirtEvent|
+    	dirtEvent.sendSynth('dj-filter' ++ ~dirt.numChannels,
+	    	// OPTIONAL
+	    	// passing this array of parameters could be left out,
+	    	// but it makes it clear what happens
+	    	[
+	    		dj: ~dj,
+	    		out: ~out
+	    	]
+    	)
+    }, { ~dj.notNil});
+
+    SynthDef("dj-filter" ++ ~dirt.numChannels, { |out, dj|
+    	var signal;
+        var lpfCutoff = (dj*2) min: 1;
```

Comment:
maybe these two would be easier to read if you write:

```
var lpfCutoff = dj.linlin(0, 0.5, 0, 1);
var hpfCutoff = dj.linlin(0.5, 1, 0, 1);
```

and you could do the mapping instantly:

```
var lpfCutoffFreq = dj.linexp(0, 0.5, 20, 10000);
var hpfCutoffFreq = dj.linexp(0.5, 1, 20, 10000);
```

then you can omit the `ControlSpec`.